tactus_data/utils/data_augment.py

```diff
@@ -113,7 +112,7 @@ def flip_h_2d(input_folder_path: Path,
                   saved
     """
     for file_name in json_name:
-        with open(str(input_folder_path) + "\\" + file_name) as file:
+        with open(str(input_folder_path / file_name)) as file:
```

Comment:
I  think open() accepts Path objects, but you could also do (input_folder_path / file_name).open()